homage-sample/src/main/res/menu/activity_main_drawer.xml

```diff
@@ -1,5 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I'm not keen on running the formatter over XML because it does stupid things like this. Not a big deal, but I'll probably tidy some of this up before/after merging.